nengo/neurons.py

```diff
@@ -89,6 +89,38 @@ def gain_bias(self, max_rates, intercepts):
 
         return gain, bias
 
+    def max_rates_intercepts(self, gain, bias):
+        """Compute the max_rates and intercepts corresponding to the given
+        gain and bias.
+
+        Note that this default implementation is very slow! Whenever possible,
+        subclasses should override this with a neuron-specific implementation.
+
+        Parameters
+        ----------
+        gain : ndarray(dtype=float64)
```

Comment:
We do in [gain_bias](https://github.com/nengo/nengo/pull/1334/files#diff-7d7af92e3e5ffca4d32c566c721ccbc4R33), which is the parallel to this function.  We could remove it from that one as well, but it should probably be the same.